src/euphorie/client/browser/training.py

```diff
@@ -238,6 +239,21 @@ def session(self):
         """Return the session for this context/request"""
         return self.context.session
 
+    def get_initial_answers(self):
+        """Pick a subset of questions, shuffle them, and initialize the answers with
+        None.
+        """
+        survey = self.webhelpers._survey
+        all_questions = api.content.find(
+            obj=survey,
```

Comment:
```suggestion
            context=survey,
```
Sorry I tricked you but the correct parameter is `context` and not `obj`: https://github.com/plone/plone.api/blob/e5f5d455a6cf8f87877c13425eff16dd41f217bf/src/plone/api/content.py#L591
:man_facepalming: